### ISSUE TITLE
Losing content indentation after @include when using sass syntax

### ISSUE THREAD

**Issue by mauricio-fleeet** [TheRealSyler/sass-formatter#9]
The formatter is clearing indentation from @includes content when using sass syntax

This code..
```sass
.some-class
    padding-bottom: 10px
    
    @include media-breakpoint-down(sm)
        padding-bottom: 0
```
It is formatted like this on save, preventing from passing @content to the @include
```sass
.some-class
    padding-bottom: 10px
    
    @include media-breakpoint-down(sm)
    padding-bottom: 0
```

**Comment by TheRealSyler** [comment]
its working as intended, i always use @include like this
```sass
@mixin PseudoCircle($color: #4afa, $width: 6px)
  content: ''
  @include Center($pos: absolute)
  transform: translate( -50%, -50% ) scale(0.97) rotate(45deg)
```

im not sure how to solve this one, because not  indenting is valid and indenting is also valid, i have to think about it, in the mean time use `///R`.
example of `///R`
```sass
.some-class
  padding-bottom: 10px

  @include media-breakpoint-down(sm)
    ///R
    padding-bottom: 0
```

**Comment by FionnT** [comment]
This is also affecting @font-face declarations: https://www.dropbox.com/s/4e92ejbe43mbh2z/Screen%20Recording%202019-12-26%20at%2017.05.26.mov?dl=0 

The formatter wasn't removing the indentations previously on either case

**Comment by TheRealSyler** [comment]
@FionnT the @font-face thing was  fixed in b4d2aaf, the formatter and the extension are not yet updated, not sure when i will do that.
